libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -844,6 +847,7 @@ class MapboxNavigation(
 
         private const val USER_AGENT: String = "MapboxNavigationNative"
         private const val THREADS_COUNT = 2
+        private const val MIN_DAYS_TO_STORE_TILES = 0
```

Comment:
@mskurydin @etl as far as I get we don't want to expose it yet. Amout of days which we don't want to update tiles if it's been updated.

Reply:
Got it. Could we update the name of the constant reflecting?

> Amout of days which we don't want to update tiles if it's been updated.

Reply:
we can expose it and leave deafult 0 in the builder

Reply:
Did we have a request for it? It's easier to expose than to hide